docs/modelserving/v1beta1/rollout/canary-example.md

```diff
@@ -0,0 +1,256 @@
+the [First Inference Service](https://kserve.github.io/website/0.8/get_started/first_isvc/) tutorial.
+
+Send more requests to the `InferenceService` to observe the 10% of traffic that routes to the new revision.
+
+## Promote the canary model
+
+If the canary model is healthy/passes your tests, you can promote it by removing the `canaryTrafficPercent` field and
+re-applying the `InferenceService` custom resource.
+
+```shell
+kubectl apply -n kserve-test -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model-2"
+EOF
+```
+
+Now all traffic goes to the revision 2 for the new model.
+
+```
+kubectl get isvc sklearn-iris
+NAME       URL                                   READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                AGE
+sklearn-iris   http://sklearn-iris.kserve-test.example.com   True           100                              sklearn-iris-predictor-default-00002   17m
+```
+
+The pods for revision generation 1 automatically scales down to 0 as it is no longer getting the traffic.
+
+```bash
+kubectl get pods -l serving.kserve.io/inferenceservice=sklearn-iris
+NAME                                                           READY   STATUS        RESTARTS   AGE
+sklearn-iris-predictor-default-00001-deployment-66c5f5b8d5-gmfvj   1/2     Terminating   0          17m
+sklearn-iris-predictor-default-00002-deployment-5bd9ff46f8-shtzd   2/2     Running       0          15m
+```
+
+## Rollback and pin the previous model
+
+You can pin the previous model (model v1, for example) by setting the `canaryTrafficPercent` to 0 for the current
+model (model v2, for example). This rolls back from model v2 to model v1 and decreases model v2's traffic to zero.
+
+Apply the custom resource to set model v2's traffic to 0%.
+
+```shell
+kubectl apply -n kserve-test -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    canaryTrafficPercent: 0
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model-2"
+EOF
+```
+
+Check the traffic split, now 100% traffic goes to the previous good model (model v1) for revision generation 1.
+
+```
+kubectl get isvc sklearn-iris
+NAME       URL                                   READY   PREV   LATEST   PREVROLLEDOUTREVISION              LATESTREADYREVISION                AGE
+sklearn-iris   http://sklearn-iris.kserve-test.example.com   True    100    0        sklearn-iris-predictor-default-00001   sklearn-iris-predictor-default-00002   18m
+```
+
+The pods for previous revision (model v1) now scales back up because 100% of the traffic now routes to it while the new
+model (model v2) scales down.
```

Comment:
yes because we are rolling back to v1 in this case.

Reply:
you can try out but I think actually v2 will not scale down because KServe defaults the minReplica to 1 and v2 is still considered as active revision.